internal/httprouter/config.go

```diff
@@ -0,0 +1,70 @@
+package router
+
+import (
+	"boilerplate/internal/framework"
+	"context"
+	"fmt"
+	"github.com/spf13/viper"
+	"go.uber.org/dig"
+	"go.uber.org/fx"
+	"go.uber.org/zap"
+	"net/http"
+)
+
+type ModuleConfig struct {
+	Port                   int `mapstructure:"HTTP_APP_PORT"`
+	RedirectTrailingSlash  bool
+	RedirectFixedPath      bool
+	HandleMethodNotAllowed bool
+	HandleOPTIONS          bool
+	NotFound               http.Handler
+	MethodNotAllowed       http.Handler
+	PanicHandler           func(http.ResponseWriter, *http.Request, interface{})
+	container              *dig.Container
+}
+
+func ModuleHooks(
+	lc fx.Lifecycle,
+	router *Router,
+	routes *framework.Routes,
+	logger *zap.Logger,
+) error {
+	lc.Append(
+		fx.Hook{
+			OnStart: func(context.Context) error {
+				router.AddRoutes(routes.GetRoutesInfo())
```

Comment:
I think that its more convenient to configure components directly in ModuleHooks

Reply:
I have not got it. Please, clarify your idea.